core/unit_test/TestSharedSpace.cpp

```diff
@@ -156,62 +155,57 @@ TEST(defaultdevicetype, shared_space) {
   unsigned int numPages                  = 100;
   size_t numBytes                        = numPages * getBytesPerPage();
 
-  // we rely on this to allocate the right amount of memory in the ALLOCATION
-  ASSERT_TRUE(numBytes % sizeof(int) == 0);
+  using DeviceExecutionSpace = Kokkos::DefaultExecutionSpace;
+  using HostExecutionSpace   = Kokkos::DefaultHostExecutionSpace;
 
   // ALLOCATION
   Kokkos::View<int*, Kokkos::SharedSpace> sharedData("sharedData",
                                                      numBytes / sizeof(int));
-  Kokkos::View<int*, Kokkos::DefaultExecutionSpace::memory_space>
-      defaultExecData("defaultExecData", numBytes / sizeof(int));
-  Kokkos::View<int*, Kokkos::DefaultHostExecutionSpace::memory_space>
-      defaultHostExecData("defaultHostExecData", numBytes / sizeof(int));
+  Kokkos::View<int*, DeviceExecutionSpace::memory_space> deviceData(
+      "deviceData", numBytes / sizeof(int));
+  Kokkos::View<int*, HostExecutionSpace::memory_space> hostData(
+      "hostData", numBytes / sizeof(int));
   Kokkos::fence();
 
   // GET DEFAULT EXECSPACE LOCAL TIMINGS
-  auto defaultExecLocalTimings = incrementInLoop<Kokkos::DefaultExecutionSpace>(
-      defaultExecData, numRepetitions);
+  auto deviceLocalTimings =
+      incrementInLoop<DeviceExecutionSpace>(deviceData, numRepetitions);
 
   // GET DEFAULT HOSTEXECSPACE LOCAL TIMINGS
-  auto defaultHostExecLocalTimings =
-      incrementInLoop<Kokkos::DefaultHostExecutionSpace>(defaultHostExecData,
-                                                         numRepetitions);
+  auto hostLocalTimings =
+      incrementInLoop<HostExecutionSpace>(hostData, numRepetitions);
 
   // GET PAGE MIGRATING TIMINGS DATA
-  std::vector<decltype(defaultExecLocalTimings)> defaultExecSharedTimings{};
-  std::vector<decltype(defaultExecLocalTimings)> defaultHostExecSharedTimings{};
+  std::vector<decltype(deviceLocalTimings)> deviceSharedTimings{};
+  std::vector<decltype(deviceLocalTimings)> hostSharedTimings{};
```

Comment:
I know they are the same type but I'd prefer 
```suggestion
  std::vector<decltype(hostLocalTimings)> hostSharedTimings{};
```